### PR TITLE
OLS-4 call for application related updates

### PR DESCRIPTION
**Pull request by malvikasharan**
## Summary

Add Open Review link and other minor fix

### CODE REVIEW

File: _includes/ols-4-timeline.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
-- **June 15, 2021**: Opening of the applications on Open Review
-
-    You can read this [Application guide](https://docs.google.com/document/d/e/2PACX-1vSsw0mbEXEvCFCqsJrrwJj6Ro41Cd90qbRkaifp9qn4TTf5sJAGAlG_R7pNgcktC_AqCzXZwiEbIzcM/pub) when preparing your application.
-
+- **June 15, 2021**: Call for Application opens, see the [guideline and template](https://github.com/open-life-science/application-forms)
```
Comment (malvikasharan) [inline _includes/ols-4-timeline.md:1]:
```suggestion
- **June 15, 2021**: Call for Application opens, see the [guidelines and templates](https://github.com/open-life-science/application-forms)
```

**Comment by malvikasharan** [comment]
Removed host section and have them merged with organizers

File: css/custom.scss [hunk 2]
```diff
@@ -330,4 +330,37 @@ $link-active: $link-hover;
 
 .peoples {
   margin-top: 1em;
-}
\ No newline at end of file
+}
+
+.button {
```
Comment (yochannah) [inline css/custom.scss:335]:
```suggestion
.button {
 a {color:#fff}
```

Hopefully better contrast for the text

File: css/custom.scss [hunk 3]
```diff
@@ -330,4 +330,37 @@ $link-active: $link-hover;
...
+   border-radius: 20px;
+   height: 50px;
+   line-height: 50px;
+   color: #FFFFFF;
+   font-family: Arial;
+   width: 281px;
+   font-size: 20px;
+   font-weight: 900;
+   padding: 12px;
+   background-color: #48B87D;
```
Comment (yochannah) [inline css/custom.scss:347]:
```suggestion
   background-color: $dark-green;
```

use variable rather than hardcode

File: css/custom.scss (hunk 2 above)
Comment (yochannah) [inline css/custom.scss]:
```suggestion
.button, a.button, .content a.button {
```
one of these rules should help override the grey text that's hard to read

**Comment by bebatut** [comment]
I made some changes to simplify the button (using existing classes from Bulma). It looks like this now: 

Screenshot 2021-06-14 at 21 27 26

I also remove some redundant "code" with funders/partners (really similar `_includes` file that I merged)

**Pull request merged**